package.json

```diff
@@ -126,9 +130,10 @@
       "babel-runtime>core-js": false,
       "eth-sig-util>ethereumjs-abi>ethereumjs-util>keccakjs>sha3": true,
       "eth-sig-util>ethereumjs-util>keccak": true,
-      "eth-sig-util>ethereumjs-util>secp256k1": true,
-      "ethereumjs-util>ethereum-cryptography>keccak": true,
-      "ethereumjs-util>ethereum-cryptography>secp256k1": true
+      "eth-sig-util>ethereumjs-util>secp256k1": false,
```

Comment:
I am not sure how install scripts work, but my hypothesis is that executables get installed to a singular location, so if one package installs an executable then you don't need another package to install the same executable. Is this true?

Reply:
I don't think so. Or at least I wouldn't assume that it works the same way across packages. I looked into a few that drop the executable in the package directory, so it would get a separate copy for each different version of that package present in the dependency tree

Reply:
Okay. I'll change this back.

Reply:
Actually, I don't think any of these need to be turned on. I just did a quick test to set all of these to `false` and ran the tests and they still work. I can do this in another PR though.